client/Windows/wf_client.c

```diff
@@ -272,6 +272,29 @@ static BOOL wf_pre_connect(freerdp* instance)
 	return TRUE;
 }
 
+static void wf_append_item_to_system_menu(HMENU hMenu, UINT fMask, UINT wID, const wchar_t* text,
+                                          void* dwItemData)
+{
+	// initial windows system item position where we will insert new menu item
```

Comment:
Change the comment to C comments, not C++

Reply:
Sorry I do not understand what this means. Do you mean comments `/* like this */` ? I changed the comment to this format.

Reply:
Yes unfortunately some (old) compiler don't like C++ style comment in plain C